time        stamp
uint16      port
TopicInfo[] topics
Remote[]    remotes

# When this message is sent to a remote,
# the remote_label is set to that remote's
# label used in the channels.
string      remote_label